app/services/breadcrumbs_options.rb

```diff
@@ -20,11 +20,11 @@ def bar_props
   private
 
   delegate :logger, :to => :Rails
-  attr_reader :controller_name, :action_name, :page_header, :options
+  attr_reader :controller, :action_name, :page_header, :options
   def index_item
     {
-      caption: _(controller_name.humanize),
-      url: try("#{controller_name}_path")
+      caption: _(controller.resource_name.pluralize.humanize),
```

Comment:
nitpick: maybe ```delegate resource_name to controller```?

Reply:
ok, added delegate to all other method we use from `controller`